common/src/test/java/com/revenuecat/purchases/common/DeviceCacheTest.kt

```diff
@@ -153,6 +153,20 @@ class DeviceCacheTest {
         assertThat(info?.requestDate?.time).isEqualTo(1565951442000L)
     }
 
+    @Test
+    fun `given a valid customer info with schema version, the JSON is parsed correctly`() {
+        val deprecatedValidCachedCustomerInfo by lazy {
+            JSONObject(Responses.validFullPurchaserResponse).apply {
+                put("schema_version", CUSTOMER_INFO_SCHEMA_VERSION)
+                put("customer_info_request_date", 1234567890L)
+            }.toString()
+        }
+        mockString(cache.customerInfoCacheKey(appUserID), deprecatedValidCachedCustomerInfo)
+        val info = cache.getCachedCustomerInfo(appUserID)
+        assertThat(info).`as`("info is not null").isNotNull
+        assertThat(info?.schemaVersion).isEqualTo(CUSTOMER_INFO_SCHEMA_VERSION)
```

Comment:
We don't use this `schemaVersion` property anywhere, so it could be deleted... But leaving it since it's public API and we want to keep existing behavior. With the previous change, this was always 0